src/dagmc/dagmcmetadata.cpp

```diff
@@ -209,32 +209,30 @@ void dagmcMetaData::parse_material_data() {
 
     // set the material value
     volume_material_property_data_eh[eh] = grp_name;
-
-    bool is_graveyard =
-        to_lower(grp_name).find(to_lower(graveyard_str)) != std::string::npos;
-    bool is_vacuum =
-        to_lower(grp_name).find(to_lower(vacuum_str)) != std::string::npos;
+    logger.message("Group name -- " + grp_name);
+    bool is_graveyard = to_lower(grp_name) == to_lower(graveyard_mat_str());
+    bool is_vacuum = to_lower(grp_name) == to_lower(vacuum_mat_str());
```

Comment:
This may be good for long term readability
```suggestion
    bool is_graveyard = (to_lower(grp_name) == to_lower(graveyard_mat_str()));
    bool is_vacuum = (to_lower(grp_name) == to_lower(vacuum_mat_str()));
```